content/en/docs/ops/common-problems/network-issues/index.md

```diff
@@ -419,8 +419,14 @@ spec:
         host: httpbin.org
 {{< /text >}}
 
-In this example, the gateway is terminating TLS while the virtual service is using TLS based routing.
-The TLS route rules will have no effect since the TLS is already terminated when the route rules are evaluated.
+In this example, the gateway is terminating TLS (the `tls.mode` configuration of the gateway is `SIMPLE`,
+not `PASSTHROUGH`) while the virtual service is using TLS based routing . The behavior of calculating routing rules
+occurs after the gateway terminates TLS, so the TLS routing rule will be no effect.
+
+{{< quote >}}
+More specifically, in this example, after TLS termination, the gateway communicates with the destination using the
+HTTP protocol instead of HTTPS(HTTP over TLS).
+{{< /quote >}}
```

Comment:
No spaces before a period, add a hyphen between TLS and based, and drop the quote which mostly repeats what is above.

```suggestion
In this example, the gateway is terminating TLS (the `tls.mode` configuration of the gateway is `SIMPLE`,
not `PASSTHROUGH`) while the virtual service is using TLS-based routing. The behavior of calculating routing rules
occurs after the gateway terminates TLS, so the TLS routing rule will have no effect, and will be over HTTP rather than HTTPS.
```

Reply:
It's ok.